### ISSUE TITLE
Add support  for training on Google's ARCO ERA5

### ISSUE THREAD

**Issue by jacobbieker** [openclimatefix/graph_weather#50]
## Detailed Description
The Google ERA5 archive is available: https://github.com/google-research/ARCO-ERA5 and goes back quite far. Its currently being expanded, but already covers years of data at the native model resolution. This is probably quicker and faster to pull data from than HuggingFace, and is already on Google Public Datasets, so should be easy to integrate. 

## Context
Its a lot of data in an accessible and publicly available place that we can try to leverage for training the graph models easily.

## Possible Implementation
A HuggingFace dataset script like for the EUMETSAT data? Or can just add Xarray opening it in this repo directly as well.

**Comment by jacobbieker** [comment]
The  HuggingFace datasets script will be hosted  here https://huggingface.co/datasets/openclimatefix/arco-era5

**Comment by ishachadalavada** [comment]
Hi, is this issue available to work on?

**Comment by jacobbieker** [comment]
Yes, I believe so, I can assign you it?

**Comment by ishachadalavada** [comment]
Yes, that sounds good!

**Comment by FilippoContessa** [comment]
@jacobbieker could you assign to me?

**Comment by jacobbieker** [comment]
@FilippoContessa yeah, I'd recommend doing this one or the other issue, which would you prefer? I can always assign you to the other one after the first one is done?

**Comment by Satvik24511** [comment]
Hi @jacobbieker , I’d like to work on this issue. My plan is to add an ArcoERA5Loader inside the graph_weather package that uses Xarray + Zarr + GCSFS to open the Google ARCO ERA5 datasets. I’ll include:

- A minimal loader class for a chosen variable (e.g. surface_pressure)

- A small test to confirm dimensions (time, latitude, longitude)

Could you please assign this issue to me so I can proceed?
